modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -258,6 +266,58 @@ export default {
     async create() {
       await this.edit(null);
     },
+    async import(action) {
```

Comment:
Ideally this method would be a `andleUtilityOperation` generic one, but since we are using specific component, managing file... it's not that easy and I didn't have enough time to manage this cleanly. So maybe we could discuss it and create ticket for it.

Reply:
per discussion this needs to be fixed before the PR lands, we arrived at a plan.

Reply:
Has been changes in `handleUtilityOperation`, now it manages new modals or confirmation modal as before, support for form should maybe be added but not sure what possibilities should be available.